src/plugins/home/public/application/components/home.js

```diff
@@ -128,41 +124,22 @@ export class Home extends Component {
       <KibanaPageTemplate
         data-test-subj="homeApp"
         pageHeader={{
-          pageTitle: <FormattedMessage id="home.header.title" defaultMessage="Home" />,
-          rightSideItems: overviewPageActions({
-            addBasePath,
-            application,
-            showDevToolsLink: true,
-            showManagementLink: true,
-          }),
+          bottomBorder: false,
+          pageTitle: <FormattedMessage id="home.header.title" defaultMessage="Welcome home" />,
         }}
         template="empty"
       >
-        {solutions.length ? (
-          <SolutionsSection
-            addBasePath={addBasePath}
-            solutions={solutions}
-            directories={directories}
-          />
-        ) : null}
-
-        <EuiFlexGroup
-          className={`homData ${
-            addDataFeatures.length === 1 && manageDataFeatures.length === 1
-              ? 'homData--compressed'
-              : 'homData--expanded'
-          }`}
-        >
-          <EuiFlexItem>
-            <AddData addBasePath={addBasePath} features={addDataFeatures} />
-          </EuiFlexItem>
-
-          <EuiFlexItem>
-            <ManageData addBasePath={addBasePath} features={manageDataFeatures} />
-          </EuiFlexItem>
-        </EuiFlexGroup>
-
-        <EuiHorizontalRule margin="xl" aria-hidden="true" />
+        <EuiSpacer size="m" />
```

Comment:
```suggestion
```

Small design nit. I would likely remove this Spacer. It adds an uneven flow between the header, the title, and the cards. This also will make it similar to the heading treatment in other spots.

I don't think this is essential, it's just something I noticed.

![image](https://user-images.githubusercontent.com/324519/129416990-460f069b-ec13-400a-8f96-2455e06866e9.png)

![image](https://user-images.githubusercontent.com/324519/129417068-acefa934-4a97-4980-b05a-6d06f7864fb7.png)